packages/std/src/results/attribute.rs

```diff
@@ -9,7 +9,7 @@ pub struct Attribute {
 }
 
 /// Creates a new Attribute.
-pub fn attr<K: ToString, V: ToString>(key: K, value: V) -> Attribute {
+pub fn attr(key: impl ToString, value: impl ToString) -> Attribute {
```

Comment:
Interesting... I didn't know this worked for inputs, only outputs. (I thought `Box<dyn ToString>` would be needed.

I had a discussion of this on some PR with Simon and we agreed this should be `Into<String>` trait to avoid a clone when a `String` is passed in. That would be aligned with `Response.add_attribute` method. We could use `impl` there as well.

Stuff for another PR, just saw the changes here and wanted to flag it.

Reply:
Yeah, I was wondering about `ToString`. I know the convention is normally to go `impl Into<String>`.

:+1:

Reply:
Happy to change that, but in a separate ticket because this will affect a lot of contract code

Reply:
> Yeah, I was wondering about `ToString`

We were young and stupid …